python/paddle/dataset/common.py

```diff
@@ -75,13 +75,13 @@ def download(url, module_name, md5sum, save_name=None):
     retry_limit = 3
     while not (os.path.exists(filename) and md5file(filename) == md5sum):
         if os.path.exists(filename):
-            print "file md5", md5file(filename), md5sum
+            print(("file md5", md5file(filename), md5sum))
         if retry < retry_limit:
             retry += 1
         else:
             raise RuntimeError("Cannot download {0} within retry limit {1}".
                                format(url, retry_limit))
-        print "Cache file %s not found, downloading %s" % (filename, url)
+        print(("Cache file %s not found, downloading %s" % (filename, url)))
```

Comment:
no need for new ```(```?

Reply:
Thanks for reviewing, I will fix it in next commit